docs/content/get-started/install-with-helm.md

```diff
@@ -223,10 +220,7 @@ The following table lists the configurable parameters of the NGINX Service Mesh
 | `mtls.upstreamAuthority` | Upstream authority settings. If left empty, SPIRE is used as the upstream authority. See [values.yaml](https://github.com/nginxinc/nginx-service-mesh/blob/main/helm-chart/values.yaml) for how to configure. | {} |
 {{% /table %}}
 
-{{<important>}}
-The `autoInjection.disabledNamespaces` configurable is deprecated and will be removed in a future release of NGINX Service Mesh.
-{{</important>}}
 
 {{<important>}}
-Top level configuration items `enabledNamespaces` and `disableAutoInjection` have been added. The `autoInjection.enabledNamespaces` and `autoInjection.disable` configurables have been deprecated and will be removed in a future release of NGINX Service Mesh.
+Top level configuration items `enabledNamespaces` and `disableAutoInjection` have been added. The `autoInjection.enabledNamespaces` and `autoInjection.disable` configurables have been removed.
 {{</important>}}
```

Comment:
I don't think we need this important notice at all.

Reply:
I'm inclined to agree. Including the notice for a future removal also runs the risk of forgetting to remove the notice when it happens. It's the kind of information that would better fit a release note.